src/html/to_html_tree.ml

```diff
@@ -125,25 +125,25 @@ struct
     -> Model.Lang.TypeExpr.t -> ('inner, 'outer) text Html.elt list
   = fun ?(needs_parentheses=false) t ->
     match t with
-    | Var s -> [Markup.Type.var ("'" ^ s)]
-    | Any  -> [Markup.Type.var "_"]
+    | Var s -> [Markup.ML.Type.var ("'" ^ s)]
```

Comment:
I guess this kind of pattern can be solved with first class modules as well? dropping the namespace and just reify a specific module implementation for each language? This solution was quicker and the least intrusive

Reply:
I don't know how you would factor this out... I was thinking of using `include` statements, but most bindings of this module are recursively `and`ed... not sure if you can override specific functions then?

Reply:
IIRC, in the diff between `to_html_tree.ml` and `to_re_html_tree.ml`, the skeleton of the functions is the same, and only some details change (what delimiter to use in records, variants, whether there is a semicolon, etc). So, we would factor out these details into `ML` and `Re` modules, and turn `To_html_tree` into a functor that contains the remaining common code.